infra/cifuzz/cifuzz.py

```diff
@@ -203,6 +209,109 @@ def run_fuzzers(fuzz_seconds, workspace, project_name):
   return True, False
 
 
+def get_project_coverage_report(project_name):
+  """Gets the report json for a specific OSS-Fuzz project from GCS.
+
+  Args:
+    project_name: The name of the relevant OSS-Fuzz project.
+
+  Returns:
+    The projects coverage json dict or None on failure.
+  """
+  project_report_url = fuzz_target.url_join(fuzz_target.GCS_BASE_URL,
+                                            PROJECT_COVERAGE_REPORT,
+                                            project_name + '.json')
+  project_cov_json = get_json_from_url(project_report_url)
+  if not project_cov_json:
+    logging.error('Could not get the coverage report json from %s.',
+                  project_report_url)
+    return None
+  if 'fuzzer_stats_dir' not in project_cov_json:
```

Comment:
I don't think this check belongs to this function, also I can hardly imagine us having such a JSON file uploaded

Reply:
Removed.